### ISSUE TITLE
work results are being blocked on www.bing.com

### ISSUE THREAD

**Issue by kflili** [abp-filters/abp-filters-anti-cv#465]
### List the website(s) where you encounter issues with ABP filters 
`www.bing.com`

### Define the problem 
We are form Bing At Work team. We are providing a work search results to our users. For data privacy reasons, we need to dynamically inject work results into the results list (ol#b_results) on client side based on ranking.
 
The following new rule will block our work-related search results to be shown in bing.com
! #CV-776 bing.com#$#freeze-element ol#b_results
 
Could you please revert this rule?

### Screenshot(s) 

mocked test results
![image](https://user-images.githubusercontent.com/19175478/94323398-814a1480-ff4a-11ea-896e-405e228b0c3d.png)

**Comment by llacb47** [comment]
You work at bing?

**Comment by kflili** [comment]
> You work at bing?

yes, BingAtWork is a project from bing.com. Now it become a work results search for all Microsoft. Its new name is Microsoft Search in Bing.

**Comment by wizmak** [comment]
> mocked test results
> ![image](https://user-images.githubusercontent.com/19175478/94323398-814a1480-ff4a-11ea-896e-405e228b0c3d.png)

@kflili I will try to fix the issue without removing the filter completely, is it possible to give me access to that mock/test account?
If not, can you send a copy of the highlighted HTML element?

**Comment by kflili** [comment]
Thanks @wizmak ! I sent you the detail info through your github email. Please feel free to let me know if you need any other information.

**Comment by wizmak** [comment]
@kflili You're welcome!
This should be fixed now, please confirm if it looks good on your end.

**Comment by kflili** [comment]
@wizmak I double checked and it's fixed now. Thanks very much!

**Comment by wizmak** [comment]
> I double checked and it's fixed now. Thanks very much!

@kflili You're welcome!

**Comment by yuchaozh** [comment]
Hi @wizmak, thanks for fixing the filter issue. Now we are seen another issue related to adBlock and adBlockPlus. 

when both adBlock and adBlockPlus are enabled. Our results in Bing.com homepage and the work tab are blocked.
![image](https://user-images.githubusercontent.com/3889738/94718362-83bfbc00-0306-11eb-96f8-ad6d686b1c6e.png)
![image](https://user-images.githubusercontent.com/3889738/94718422-93d79b80-0306-11eb-8687-1ba242841241.png)

But if we only keep one of them enabled. Our content is not removed.
enable adblock but disable adblockPlus
![image](https://user-images.githubusercontent.com/3889738/94718298-6a1e7480-0306-11eb-977b-1eb84b8dc56a.png)
enable adblockPlus but disable adblock
![image](https://user-images.githubusercontent.com/3889738/94718598-cd100b80-0306-11eb-8a3e-d7f6856f5673.png)

Do you know what is causing this issue? Do you know how to resolve it?
Thanks!
